cmake/public/ComputeLibrary.cmake

```diff
@@ -21,10 +21,10 @@ if("${ACL_VERSION_FILE}" STREQUAL "")
   message(WARNING "Build may fail: Could not determine ACL version (minimum required is ${ACL_MINIMUM_VERSION})")
 else()
   file(READ ${ACL_VERSION_FILE} ACL_VERSION_STRING)
-  string(REGEX MATCH "v([0-9]+\\.[0-9]+)" ACL_VERSION ${ACL_VERSION_STRING})
+  string(REGEX MATCH "v([0-9]+\\.[0-9]+)" ACL_VERSION "${ACL_VERSION_STRING}")
```

Comment:
This seems unrelated to the PR in question

Reply:
This file is used to build ACL which is needed for this patch and on my version of CMake this is the only way to resolve a "syntax error" caused by an unquoted expansion of that variable.